src/Spice86/ViewLocator.cs

```diff
@@ -18,6 +18,9 @@ internal sealed class ViewLocator : IDataTemplate {
     /// <returns>The corresponding View, or a TextBlock with the "Not Found" message if a match wasn't found.</returns>
     public Control Build(object? data) {
         string? name = data?.GetType().FullName?.Replace("ViewModel", "View");
+        if(name == "Spice86.Views.MemoryView") {
+            name = "Spice86.Views.MemoryView";
+        }
```

Comment:
this confuses me :)

Reply:
The Stack memory view doesn't exist yet. I just reuse the memory view since the stackMemoryViewModel inherits from the MemoryViewModel and just adds a tiny amount of behavior on top.

Reply:
That said the if is wrong.

Reply:
Fixed.